gitlab/v4/objects/boards.py

```diff
@@ -48,7 +48,7 @@ class ProjectBoardListManager(CRUDMixin[ProjectBoardList]):
     _obj_cls = ProjectBoardList
     _from_parent_attrs = {"project_id": "project_id", "board_id": "id"}
     _create_attrs = RequiredOptional(
-        exclusive=("label_id", "assignee_id", "milestone_id")
+        exclusive=("label_id", "assignee_id", "milestone_id", "iteration_id")
```

Comment:
According to the docs I don't see that `iteration_id` is mutually exclusive. Can you confirm?

https://docs.gitlab.com/api/boards/#create-a-board-list